task2/js/app.js

```diff
@@ -18,9 +18,9 @@ document.addEventListener('DOMContentLoaded', () => {
         const zombieElement = document.createElement('div');
         zombieElement.classList.add('zombie-item', zombie.icons);
 
-        zombieElement.addEventListener('click', (event) => {
-            event.target.classList.add('zombie-dead');
-        });
+        zombieElement.onclick = function() {
```

Comment:
nope, use "addEventListener" instead
P.S.: you can't implement this solution without understanding [[ENVIRONMENT]]